src/node_alive/autostarter.py

```diff
@@ -0,0 +1,119 @@
+# ROS
+import os
```

Comment:
os is not a ros import

Reply:
replaced it in a1ef39d19ac8416def3517506f5d571fcb9e1645